extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -828,6 +840,10 @@ async function processAzureAccountField(context: AzureAccountFieldContext): Prom
 	}, 0);
 }
 
+function getAccountDisplayString(account: azdata.Account) {
+	return `${account.displayInfo.displayName} (${account.displayInfo.userId})`;
```

Comment:
make sure to test the live account as well, I think only showing the email address is enough.

Reply:
We should generally show the ID as well - email address isn't necessarily unique

Reply:
This is not introduced by this change. For the accounts that I have tested with account.displayInfo.userId and account.key.accountId are the same value, so including both does not make sense. Also, it would seem that we should only include fields account.displayInfo object in what is displayed to the user. Since userId is the same as accountId, we are already displaying to user what Charles has suggested. We can do any improvements as necessary in the future. 

@alanrenmsft  - if you have concerns about the correctness of this code for live account then kindly open an issue and I will get that prioritized, investigated, and tackled. The changes here do not affect the correctness or incorrectness of the live accounts, so I would like to get this fix in and tackle that separately.